spec/support/sidekiq.rb

```diff
@@ -0,0 +1,5 @@
+require "sidekiq/testing"
```

Comment:
Is there a strong reason to move this to a separate file rather than rails_helper? Beyond my desire to consolidate, I've generally seen these included in the rails_helper file.

Also goes for simplecov and vcr.